src/amo/css/AddonReview.scss

```diff
@@ -0,0 +1,52 @@
+@import "~core/css/inc/vars";
+
+.AddonReview {
+  text-align: center;
+
+  h2 {
+    font-size: $font-size-l;
+    margin-bottom: 0.3em;
+  }
+
+  p {
+    margin: auto;
+    margin-bottom: 2em;
+    width: 80%;
```

Comment:
I think it would make sense for the description and comment box to both be `width: 90%` or have a static margin or 15px/1em/1rem or something. Possibly move the margin up to the parent and let these take the full space.